lib/auth/managed_authenticator.rb

```diff
@@ -121,6 +121,8 @@ def after_create_account(user, auth)
 
     retrieve_avatar(user, association.info["image"])
     retrieve_profile(user, association.info)
+
+    auth.apply_associated_attributes! if auth.respond_to?(:apply_associated_attributes!)
```

Comment:
Because some of the existing specs (not in this PR) use simple hashes instead of authtokens which causes an error here. But looking at it now, I should probably just update those existing tests.